validation/td-validation.ttl

```diff
@@ -1028,32 +1024,29 @@
           that accepts JSON, but returns an image. In such a case,
           the optional <code>response</code> name-value pair can
           describe the content type of the expected response.
-          <span class="rfc2119-assertion" id=
-          "td-expectedResponse-contentType">If the content type of
-          the expected response differs from the content type of
-          the form, the <code>Form</code> instance <em class=
-          "rfc2119" title="MUST">MUST</em> include a name-value
-          pair with the name <code>response</code>.</span> 
-          For instance, an <code>ActionAffordance</code> could only
-          accept <code>application/json</code> for its input data,
-          while it will respond with an <code>image/jpeg</code>
-          content type for its output data. In that case the
-          content types differ and the <code>response</code>
-          name-value pair has to be used to provide response
-          content type (<code>image/jpeg</code>) information to the
-          <a href="#dfn-consumer" class="internalDFN"
-          data-link-type="dfn">Consumer</a>.</p>
+          <span class="rfc2119-assertion" id="td-expectedResponse-contentType">
+            As there is no default value defined, the content type of additional
+            responses MUST be explicitly specified.
+          </span>
+          That means that the lack of a content type value implies that the 
```

Comment:
I would be more specific/assertive here: When there is payload in the response, the `contentType` field MUST be omitted from the expected response.

Reply:
This would allow us to test both cases easily. Having `contentType` or not

Reply:
> When there is payload in the response, the `contentType` field MUST be omitted from the expected response.

Hmm, do you mean when there is _no_ payload, the field must be omitted? That would sound good to me :)

Reply:
Yes... A very well placed "typo" from my side :D